src/Symfony/Bundle/FrameworkBundle/Command/EventDispatcherDebugCommand.php

```diff
@@ -83,10 +88,18 @@ protected function execute(InputInterface $input, OutputInterface $output)
     /**
      * Loads the Event Dispatcher from the container.
      *
+     * @param string $serviceId
+     *
      * @return EventDispatcherInterface
      */
-    protected function getEventDispatcher()
+    protected function getEventDispatcher($serviceId)
```

Comment:
Technically, this is a BC break. Not sure why `getEventDispatcher()` was defined protected :/